wp/sections/feature.tex

```diff
@@ -0,0 +1,75 @@
+\par As described above, diKTat is very configurable and user-friendly. But these are not all of its advantages and features. Below will be presented and described unusual and important killer-feature diKTat.
+
+\subsection{Configuration file}
+\par
+It's worth starting with the configuration files. This is a file in which the user can manually turn rules on and off or configure the rules settings. Below is one of the rules in the configuration file.
+\begin{center}
+- name: FILE\underline{ }IS\underline{ }TOO\underline{ }LONG\\
+	enabled: true\\
+	configuration:\\
+	maxSize: '2000'\\
+	ignoreFolders: ' '\\
+\end{center}
```

Comment:
Is everything okay with indentation here? Github shows the text as if configuration flags are on the same level as other fields.